lms/djangoapps/teams/models.py

```diff
@@ -101,6 +101,8 @@ class CourseTeam(models.Model):
 
     .. no_pii:
     """
+    def __unicode__(self):
+        return "This model represents team related info"
```

Comment:
These `__unicode__` methods should return a string representation of an _instance_ of the class, not a description of the class itself. For example, a reasonable string representation of some course team might be `<CourseTeam id="assignment-team-4">`

Here's an example from a different model: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/program_enrollments/models.py#L78-L79

(Nevermind that it's `__str__` instead of `__unicode__`, we can take care of that later)